pkg/model/resources/nodeup.go

```diff
@@ -142,12 +130,8 @@ function download-release() {
     ;;
   esac
 
-  # In case of failure checking integrity of release, retry.
   cd ${INSTALL_DIR}/bin
-  until try-download-release; do
-    sleep 15
-    echo "Couldn't download release. Retrying..."
-  done
+  try-download-release
```

Comment:
`try-download-release` is doing very little now. Maybe just drop it and move what's left to `download-or-bust`.

Reply:
Probably a followup PR. I was thinking of having `download-or-bust` start with an existence-and-checksum test in case someone bakes the right nodeup into the AMI.

Reply:
That would be nice.